core/src/test/scala/com/microsoft/azure/synapse/ml/recommendation/RankingAdapterSpec.scala

```diff
@@ -15,6 +17,19 @@ class RankingAdapterSpec extends RankingTestBase with EstimatorFuzzing[RankingAd
 
   override def modelReader: MLReadable[_] = RankingAdapterModel
 
+  override def assertDFEq(df1: DataFrame, df2: DataFrame)(implicit eq: Equality[DataFrame]): Unit = {
+    def prep(df: DataFrame) = {
+      // sort rows and round decimals before compare two dataframes
```

Comment:
With the goal of making sure that we're not hiding a real issue by changing tests, are we sure it's accurate to sort and round?
Why did it work before when it was unsorted and unrounded?